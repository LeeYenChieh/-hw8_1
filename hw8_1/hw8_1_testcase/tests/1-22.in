zuJwjm4YsDpCpq4zBvsC8w0vnK6GNvK7uZENfqK7AMur 7aRdd9x2NGxWHpQDxXqEthz1h1XIlbRWAj8yvclt4sGKOg5aYD
ZIyYjayLMmpo2E2bXT7ipOu65 WpvpnxECRdCagBVRqUS1FJ8TAI83WtUxgoPFDzFWuiVJfE
bwu0 1h6njgWR5Ynu26OBl2oe9Umg9
ZIyYjayLMmpo2E2bXT7ipOu65 display
vB6WAglGaw3TP02xwV8wkin 3V86Ngz1CVkdq8AFNntFjUBuYGNv26kWO
bwu0 display
1afLIlTOMpOwjUHvJDwEHUCr3oq8ZY5 sRjAw321zlEyWhnR2byDOLWETSo8nlTqZ
Xu4lbnCOzM3oZQ1kezHfdH7nvXQt CIf6R9KzTvv9mdH
oMD JEgqpNlDRcvZFra5w8fM9hwyCcQPENZ53e4Udx44PIkIK7lATX1TlSRcJZ
IC5YBLb6Q0DQaqTfBWdW96iaw kpxKh2FFIPmK5oEenr3HDqT
