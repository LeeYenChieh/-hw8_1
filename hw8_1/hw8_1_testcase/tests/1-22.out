WpvpnxECRdCagBVRqUS1FJ8TAI83WtUxgoPFDzFWuiVJfE 
1h6njgWR5Ynu26OBl2oe9Umg9 
